state/execution_test.go

```diff
@@ -206,10 +206,11 @@ func TestFinalizeBlockValidators(t *testing.T) {
 		desc                     string
 		lastCommitSigs           []types.ExtendedCommitSig
 		expectedAbsentValidators []int
+		shouldHaveTs             bool
```

Comment:
The linter's complaining about this field name - is the "Ts" meant to represent "timestamp"?

Reply:
Yes. What do you suggest?

Reply:
The linter was recommending `shouldHaveTS`, but any of those should do.